languages/woocommerce-related-accessories-de_DE.po

```diff
@@ -31,6 +31,22 @@ msgstr ""
 "die Plugins WooCommerce, ACF Pro sowie YITH Quick View installiert und "
 "aktiviert sind."
 
+#: src/AdminBacklinks.php:21
+msgid "Load"
+msgstr "Laden"
+
+#: src/AdminBacklinks.php:33
+msgid "Loading"
+msgstr "Wird geladen"
+
+#: src/AdminBacklinks.php:53
+msgid "Related Accessories Backlinks"
+msgstr "Zubehör Backlinks"
```

Comment:
Sorry, just noticed this only after submitting - can you incorporate the translation and update .mo file?
```suggestion
msgstr "Zubehör Rückverlinkungen"
```